src/commons/api/history.rs

```diff
@@ -720,6 +744,19 @@ impl fmt::Display for StorableCaCommand {
                 write!(f, "Automatically re-issue objects before they would expire")
             }
 
+            // ------------------------------------------------------------
+            // ASPA Support
+            // ------------------------------------------------------------
+            StorableCaCommand::AspasUpdate { updates } => {
+                write!(f, "{}", updates)
+            }
+            StorableCaCommand::AspasUpdateExisting { customer, update } => {
+                write!(f, "update ASPA for customer ASN: {} {}", customer, update)
```

Comment:
Here you refer to ASN in the message, while above you use AS in messages. Should they be consistent?

Reply:
trying to use AS consistently